packages/core/src/configuration.ts

```diff
@@ -49,6 +49,10 @@ export interface UserConfiguration {
   internalMonitoringEndpoint?: string
   logsEndpoint?: string
   rumEndpoint?: string
+
+  clientService?: string
+  clientEnv?: string
+  clientVersion?: string
```

Comment:
we should probably remove the `client` prefix for the API.
For example, `service` will be used as `service` and not `clientService`.
It will also be consistent with current tracer API: https://datadoghq.dev/dd-trace-js/